### ISSUE TITLE
Create a video of ONIX installation and an AI transaction demo

### ISSUE THREAD

**Issue by vbabuEM** [beckn/beckn-onix#15]
A video of one full demo installation including

1. AI Bot interacting with the new network to complete a transaction
2. Voiceover should be transcribed
3. It can be around 8 minutes

**Comment by vbabuEM** [comment]
Upon discussion with Sujith on March 28 at FIDE office, it was decided that AI will not be part of this demo. Instead it will be a retail application (derived from industry 4.0) that will be used for it.

**Comment by vbabuEM** [comment]
The following is the link to the video created on March 30
https://drive.google.com/file/d/1ms1tEZ5WmqJkRi-OWf89gdzYcDyFQVlt/view?usp=drive_link

**Comment by vbabuEM** [comment]
No review comments upto now. Closing this issue.
